app/ui/browser/actions/external.js

```diff
@@ -227,7 +228,7 @@ export function close() {
  * Perform the search if the user just pressed return
  */
 export function inPageSearch(ev) {
-  if (ev.keyCode === 13) {
+  if (ev.keyCode === KEY.ENTER) {
```

Comment:
Is that easier? Are there going to be cross platform differences with using that? (I honestly don't know)

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#Key_values